src/main/java/models/Game.java

```diff
@@ -42,7 +42,15 @@ public void buildDeck() {
         }
     }
 
-
+    public void buildSpanishDeck() {
+        //1 for Ace, 8 for Jack, 9 for Knight, 10 for King
+        for(int i = 1; i < 11; i++){
+            deck.add(new Card(i,Suit.Swords));
```

Comment:
This should be correct, I'll add name functionality and a new constructor that will be used.